cypress/integration/use-cases/donate.spec.js

```diff
@@ -0,0 +1,43 @@
+describe('Using the donate function', () => {
+    it('lets users donate via bank transfer', () => {
+        cy.visit('/donate');
+        cy.contains('10 €').click();
+        cy.get('#donation-widget-amount').should('have.value', 10);
```

Comment:
Let's also test that the text to the right ("You will be billed 10.00 € once.") contains the correct value.